scripts/jest-package.js

```diff
@@ -7,18 +9,20 @@ import runInPackage from './utils/base-package';
  * @returns {Promise}
  */
 export default function jestPackage(pkgName, pkgPath) {
-  return runInPackage({
```

Comment:
Oof, yea sorry about that one. There's a bit of goofiness with that when I moved the other scripts to be more synchronous.